hightime/_timedelta.py

```diff
@@ -137,6 +138,16 @@ def total_seconds(self):
             + (self.yoctoseconds / 10 ** 24)
         )
 
+    def precision_total_seconds(self):
+        """Precise total seconds in the duration."""
+        return Decimal(
+            (self.days * 86400)
+            + self.seconds
+            + Decimal(self.microseconds) / Decimal(10**6)
+            + Decimal(self.femtoseconds) / Decimal(10**15)
+            + Decimal(self.yoctoseconds) / Decimal(10**24)
+        )
```

Comment:
When I was drafting the "please add this test for the situation that caused this PR" comment, I found out that will still not quite meet that case.

(this does also require doing an `import decimal` above)
```suggestion
        """Precise total seconds in the duration.
        
        Note: up to 64 significant digits are used in computation.
        """
        with decimal.localcontext() as ctx:
            ctx.prec = 64
            return Decimal(
                (self.days * 86400)
                + self.seconds
                + Decimal(self.microseconds) / Decimal(10**6)
                + Decimal(self.femtoseconds) / Decimal(10**15)
                + Decimal(self.yoctoseconds) / Decimal(10**24)
            )
```

Reply:
Thanks a lot for the suggestion, it's updated accordingly.